keg-client/src/Taps/TapsPage.tsx

```diff
@@ -54,6 +54,27 @@ class TapsPage extends React.Component<any, AppState> {
     if (this.socket) this.socket.close();
   }
 
+  async voteHandler(id: string, isUpVote: boolean) {
```

Comment:
If you define this function as

```ts
voteHandler = async (id: string, isUpvote: boolean) => {
...
}
```

you won't need to call .bind in the render function. That will improve performance. 

https://www.freecodecamp.org/news/this-is-why-we-need-to-bind-event-handlers-in-class-components-in-react-f7ea1a6f93eb/
https://www.freecodecamp.org/news/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56/